Typo in `XChaChaIETF.stream()`

### Zig Version

` >= 0.12.0`  

### Steps to Reproduce and Observed Behavior

Application doesn't compile when using `XChaChaIETF(N).stream()`.

```
src/chacha.zig:257:34: error: expected 5 argument(s), found 4
            ChaChaIETF(rounds_nb).xor(out, counter, extended.key, extended.nonce);
            ~~~~~~~~~~~~~~~~~~~~~^~~~
src/chacha.zig:211:13: note: function declared here
        pub fn xor(out: []u8, in: []const u8, counter: u32, key: [key_length]u8, nonce: [nonce_length]u8) void {
        ~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

### Expected Behavior

The application compiles.

The problem is a [typo](https://github.com/ziglang/zig/blob/5f589562646b972e4f177d60c7da37e872991d29/lib/std/crypto/chacha20.zig#L635). `XChaChaIETF.stream()` should call `ChaChaIETF.stream()` instead of `ChaChaIETF.xor()`.

--- a/lib/std/crypto/chacha20.zig
+++ b/lib/std/crypto/chacha20.zig
@@ -632,7 +632,7 @@ fn XChaChaIETF(comptime rounds_nb: usize) type {
         /// Write the output of the XChaCha20 stream cipher into `out`.
         pub fn stream(out: []u8, counter: u32, key: [key_length]u8, nonce: [nonce_length]u8) void {
             const extended = extend(key, nonce, rounds_nb);
-            ChaChaIETF(rounds_nb).xor(out, counter, extended.key, extended.nonce);
+            ChaChaIETF(rounds_nb).stream(out, counter, extended.key, extended.nonce);
         }
     };
 }

